packages/BetterPhpDocParser/Printer/PhpDocInfoPrinter.php

```diff
@@ -82,7 +78,8 @@ public function __construct(
         private DocBlockInliner $docBlockInliner,
         private RemoveNodesStartAndEndResolver $removeNodesStartAndEndResolver,
         private ChangedPhpDocNodeVisitor $changedPhpDocNodeVisitor,
-        ChangedPhpDocNodeTraverserFactory $changedPhpDocNodeTraverserFactory
+        ChangedPhpDocNodeTraverserFactory $changedPhpDocNodeTraverserFactory,
+        private EditorConfigConfiguration $editorConfigConfiguration
```

Comment:
hmm my goal is/was to use/re-use the editor configuration of the project rector is invoked on. I just realized that the editor configuration currently seems to be only available within the `Rector\FileFormatter` scope.

it seems first there is some groundwork required to make the editor config available for use in rectors/packages.. what do you guys think?

@sabbelasichon it seems your are the expert in this area?

Reply:
may I ping you again @sabbelasichon ? Could you forsee how a integration of the editor config class could/should look like, so we can also re-use this functionality on a rector rule level in a more convenient way?

Reply:
Of course. Sorry. Totally overseen.

Reply:
@staabm Is it not possible to inject the EditorConfigParser class and then call the method extractConfigurationForFile with either the file from the class CurrentFileProvider or directly from the property $file within a class extending the AbstractRector? Just a guess.

Reply:
Maybe i don´t really understand your intention / use case in mind. Maybe you could explain it in more detail

Reply:
@sabbelasichon the point/goal of the PR is to get consistent newline characters in generated phpdocs, which means using the newline-style of a possible existent `.editorconfig` file.

this has the additional benefit, that for rector-src itself we would get consistent line-endings, because of the `.editorconfig` within the rector codebase itself.

therefore this would reduce differences in newline character handling between windows/linux CI builds and ease testing

Reply:
so after all: make `PhpDocInfoPrinter` use the newline char defined in the `.editorconfig´ of the project rector is beeing ran on